lib/units/storage/plugins/apk/task/manifest.js

```diff
@@ -1,7 +1,16 @@
 var ApkReader = require('@devicefarmer/adbkit-apkreader')
+var IpaReader = require('../../../../../util/ipareader')
 
 module.exports = function(file) {
-  return ApkReader.open(file.path).then(function(reader) {
-    return reader.readManifest()
-  })
+  if(file.path.endsWith('.apk')){
```

Comment:
Formatting seems to be inconsistent here. Taking into account other files it should be rather:
`if (file.path.endsWith('.apk')) {`

Reply:
I'll change this. The style in the code is agreeably to have space between 'if and (', and this particular style difference is not something I care significantly about either way so I'm okay with sticking to the spaced style.